<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGroupDialog</class>
 <widget class="QDialog" name="EditGroupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование группы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QLabel" name="roleLabel">
     <property name="text">
      <string>Роль</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="parenLabel">
     <property name="text">
      <string>Родитель</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="parentBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="groupLabel">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="groupEdit">
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string>Введите новое название группы ...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Название группы должно быть уникальным!</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="rolesBox"/>
   </item>
   <item row="4" column="3">
    <widget class="QTextEdit" name="descriptionEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditGroupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditGroupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
